### PR TITLE
form validation fix

### PR DESCRIPTION
**Pull request by Dpschu2**
#### What?

Adding name field validation message for product reviews form.

#### Screenshots

Added name field validation

### CODE REVIEW

**Comment by bc-yevhenii-buliuk** [comment]
@Dpschu2 please add a description to the PR and a corresponding screenshot/video confirming the fix. Thanks!

**Review by bc-yevhenii-buliuk — APPROVED** [review]

**Comment by Dpschu2** [comment]
Description updated. How do you squash commits?

**Comment by bc-yevhenii-buliuk** [comment]
> Description updated. How do you squash commits?
> 
> Here is what I tried. Did that work? image

yep, you can use `git rebase -i HEAD~3` and after that in editor use commands `pick` and `squash` to define main commit. As result you will get one commit instead three.

**Comment by Dpschu2** [comment]
I originally did that by making the first one 'pick' and the last 2 'squash'. Doing it now gives me commits from past PRs:
image

**Comment by Dpschu2** [comment]
Here is what I did:
Screenshot 2024-07-16 at 12 53 14 PM

image

**Review by yurytut1993 — APPROVED** [review]

**Review by bc-alexsaiannyi — APPROVED** [review]

**Review by bc-vlad-dlogush — APPROVED** [review]

File: assets/js/theme/product/reviews.js [hunk 1]
```diff
@@ -89,6 +89,10 @@ export default class {
             selector: '[name="revtitle"]',
             validate: 'presence',
             errorMessage: safeString(this.context.reviewSubject),
+        }, {
+            selector: '[name="revfromname"]',
```
Comment (bc-azvierieva) [inline assets/js/theme/product/reviews.js:93]: @Dpschu2 I think you have meant `[name="revformname"]`, not `[name="revfromname"]` ?

Comment (bc-azvierieva) [reply assets/js/theme/product/reviews.js:93]: already got answer for this, all is good 👍

**Review by bc-azvierieva — APPROVED** [review]

**Comment by bookernath** [comment]
@Dpschu2 do you mind doing a rebase? I tried to do it for you but I don't have access to your fork. This is good to go after rebase.

**Pull request merged**